mr/src/test/java/org/apache/iceberg/mr/hive/HiveIcebergStorageHandlerTestUtils.java

```diff
@@ -36,9 +36,9 @@
       new FileFormat[] {FileFormat.AVRO, FileFormat.ORC, FileFormat.PARQUET};
 
   static final Schema CUSTOMER_SCHEMA = new Schema(
-          optional(1, "customer_id", Types.LongType.get()),
-          optional(2, "first_name", Types.StringType.get()),
-          optional(3, "last_name", Types.StringType.get())
+          optional(1, "customer_id", Types.LongType.get(), "This is an ID"),
+          optional(2, "first_name", Types.StringType.get(), "This is first name"),
+          optional(3, "last_name", Types.StringType.get(), "This is last name")
```

Comment:
do we have to have a comment for all of columns?
shall we test  at least one without a comment?